src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -1413,7 +1415,10 @@ class BookingService(
     booking.departureDate = dateTime.toLocalDate()
     updateBooking(booking)
 
-    cas3DomainEventService.savePersonDepartedEvent(booking)
+    when (findExistingDeparturesForBooking.isNullOrEmpty()) {
```

Comment:
Using the suggestion above, this would become:

```suggestion
    when (isFirstDeparture) {
```

Reply:
same as above

Reply:
done